book/website/collaboration/ppie-events/ppie-events-comms.md

```diff
@@ -0,0 +1,362 @@
+
+(cl-ppie-events-comms-timeline)=
+## When to send Core Communications
+The timeline for planning a communications schedule for your event will vary greatly depending on the size and type of event. 
+
+When planning an event attended by people with lived experience, please be mindful that many people with lived experience might require information with more notice in order to plan their attendance.
+
+The more advance notice you can give to people, the more flexibility people have to organize and make arrangements, and the more likely they will be to attend your event.
+
+(cl-ppie-events-save-the-date)=
+### Save the Date
+Consider how early you will be able to send out a "Save the Date" for your event and when you will be able to finalize accessibility information and a reimbursement policy. 
+
+For people with lived experience who are attending your event, Save the Dates serve to give the advance notice required for attendees to plan and give notice that they will be attending your event.
+For example, many professions require at least 2 weeks advance notice to process and approve leave to support a person taking time off work to attend your event, or people with caregiving responsibilities may need to arrange alternative care to support their attendance.   
+
+Additionally, many people with lived experience of disabilities or medical conditions need to carefully plan their attendance as part of managing their condition, especially for in-person events. 
+This can be due to medication requirements, and managing conditions that involve fatigue or overstimulation as well as figuring out a plan for how they will manage physical barriers to arriving at an in-person event. 
+
+Providing clear and timely information will help your attendees with lived experiences plan their attendance safely, and in a way that supports them engaging fully in your event. 
+This will also be supportive of attendees with busy schedules who may be booked out significantly in advance.
+
+Your "Save the Date" should include an overview of honorarium or reimbursement support, the date, and the approximate location of the event such as Zoom, or the name of the city where the event will take place. 
+It should also reference how long the event is likely to be, for example, if it's a multi-day event or a few hours. 
+
+By providing guidance along with a "Save the Date", your attendees would have appropriate time to be able to make arrangements, and also be able to make an informed choice about if they are able to attend. 
+
+Consider sending out different forms of a "Save the Date" to support how different people use and interact with online calendars. 
+For example, depending on your attendees you may need to:
+* send out an online calendar placeholder,
+* send an email invitation written in plain text with formatted headings,
```

Comment:
I believe that it's actually a combo of both. 

Formatted headings meaning that instead of bolding and increasing the font size, the actual formatting is used to register a heading, and then plain text referring to both plain language, and also using markup(?) plain text. 

We can add more information/clarification on this.

Reply:
Yeah, I think the challenging bit is that those are (partially) mutually exclusive. Formatted headings means non-plain text. Maybe like this for now? 

```suggestion
* send an email invitation written in plain language with simple formatted headings,
```